maintainers/maintainer-list.nix

```diff
@@ -4887,6 +4887,15 @@
       fingerprint = "5214 2D39 A7CE F8FA 872B  CA7F DE62 E1E2 A614 5556";
     }];
   };
+  gp2112 = {
```

Comment:
Please, add yourself to maintainer-list as separate commit (`maintainers: add gp2112`).

Reply:
okay, [solved](https://github.com/NixOS/nixpkgs/pull/189887/commits/b6a94490a4f05ea9a80f897de5e89cc179d440be)